spec/requests/resources_spec.rb

```diff
@@ -32,4 +32,14 @@
       end
     end
   end
+  context 'show' do
+    let!(:resources) { create_list :resource, 4 }
+    let!(:resource_a) { create :resource, name: 'a' }
+
+    it 'returns specific resource' do
+      get "/resources/#{resource_a.id}"
+      returned_ids = response_json['id']
+      expect(returned_ids).to match(resource_a.id)
```

Comment:
Was thinking we could make sure that the child resources are rendered as well. Could be:

```ruby
expect(response_json).to match(
  'id' => resource_a.id,
  'addresses' => Array,
  'schedule' => Hash,
  'phones' => Array
)
```

You could make the assertion stronger, but that at least captures the presence of nested resources.